.github/workflows/scripts/itskip_disable_check.py

```
@@ -0,0 +1,157 @@
+
+    Returns:
+        bool: True if it.skip statement is found, False otherwise.
+    """
+    # Initialize key variables
+    eslint_disable_pattern = re.compile(
+        r"\s+it\.skip\(",
+        re.IGNORECASE | re.DOTALL,
+    )
+
+    try:
+        with open(file_path, "r", encoding="utf-8") as file:
```

Ruff (0.14.3)

Unnecessary mode argument

Remove mode argument

(UP015)